sopa/io/reader/wsi.py

```diff
@@ -12,19 +12,20 @@
 
 
 def wsi(
-    path: str | Path, chunks: tuple[int, int, int] = (3, 256, 256), as_image: bool = False
+    path: str | Path, chunks: tuple[int, int, int] = (3, 256, 256), as_image: bool = False, backend: str = "openslide"
```

Comment:
Since openslide is complex to install (contrary to tiffslide that is inside the 'wsi' extra), shouldn't we have tiffslide as a default?

Reply:
sure we can select the tiffslide as default